book/docs/Open_data_code.ipynb

```diff
@@ -2,21 +2,77 @@
  "cells": [
```

Comment:
<p>Should we point people to and example of parsing config files? It's not always obvious how to do this, what library to use, etc...</p><p> </p><p>Spell our PR - on other platforms outside of Github they have different names, e.g. Gitlab users know it as a MR (merge request). Also the text here doesn't make it clear that this means a Peer review of your code. Do we cover breaking your PRs into manageable chunks (link back to the SRP) &lt;- a good PR only tackles 1 issue at a time (also use issue tracking to track the work).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/scaling_science/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>On PRs: see the above section "14. Contribute quality pull requests:" in which your comments have mostly already been addressed.</p>

Reply:
<p>As for config files, we can look at including reference to some tools. I am thinking things like Hydra for ML. Any suggestions you might have?</p>

Reply:
Added some basic parser references for use with Python.